docs/src/operate/registration/http_api.md

```diff
@@ -83,158 +83,79 @@ GET /v1/entities/{topic-id}
 curl http://localhost:8000/te/v1/entities/device/child01
 ```
 
-## Query entities
```

Comment:
The diff of this file is a bit weird. All I did was to move this `Query entities` section to the end of this file so that the CRUD operations are placed together. This `Query entities` section was quite big and in the middle of them earlier, placed along with `Fetch entities`(as querying is also another kind of READ operation).